client/api/auth.js

```diff
@@ -0,0 +1,10 @@
+import request from 'superagent'
+
+const url = 'http://localhost:3000'
```

Comment:
Please put the api here `const url = 'http://localhost:3000/api/v1/auth` as any other routes for this api will use the same url.